op-guide/migration.md

```diff
@@ -201,6 +201,23 @@ mysql> select * from t2;
 +----+------+
 ```
 
+### 3. Best practice
+In order to migrate data quickly (especially huge amount of data), you can refer to the following recommendations.
+
+- make dump data file as small as possible, it is best not to exceed 64M (parameter -F 64)
+- loader's parameter -t can be evaluated according to the number and load of tikv instances. For example, there are three tikv instances, -t can be set to 3 * (1 ~ n); while load of tikv is too high and the loader's log appears a large number of `backoffer.maxSleep 15000ms is exceeded`, it's better to decrease -t ; otherwise increase it.
```

Comment:
it is a summary of tikv's cpu, io utilization and etc.  Do you think I need to add this note? Or other more accurate description？ Maybe  `backoffer.maxSleep 15000ms is exceeded` is enough to describe tikv is busy